lib/adequate_crypto_address/aave.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module AdequateCryptoAddress
+  class Aave < Eth
+    ADDRESS_TYPES = { prod: %w[], test: %w[] }.freeze
```

Comment:
What is the purpose of the empty arrays?

Reply:
I thought we'd have different addresses in prod than when in test, but maybe I won't need to differentiate if we're only validating in prod 🤔